src/Stratis.Bitcoin.Features.Dns/DnsSeedMasterFile.cs

```diff
@@ -48,7 +131,13 @@ public void Load(Stream stream)
         /// <param name="stream">The stream to write the masterfile to.</param>
         public void Save(Stream stream)
         {
-            // TODO
+            Guard.NotNull(stream, nameof(stream));
```

Comment:
Can you use FileStorage class for that, under Stratis.Bitcoin.Utilities, or is it not appropriate?
Same question for Load(Stream) above.

Reply:
We need to control the serialization and deserialization of the IResourceRecords which are from the DNS nuget package, via a JsonConverter, so its better to receive the stream. Also the load and save get a stream, making them easier to unit test without relying on a file system.

Reply:
fair enough :-)